src/routes_student.py

```diff
@@ -25,6 +25,19 @@ def student_get_course(netid, cid):
 
 			course = db.get_course(cid)
 			assignments = db.get_assignments_for_course(cid)
+			now = util.now_timestamp()
+
+			for assignment in assignments:
+
+				num_extension_runs = get_active_extensions(cid, assignment["assignment_id"], netid, now)
+				num_available_runs = get_available_runs(cid, assignment["assignment_id"], netid, now)
+				total_runs = num_extension_runs[1] + num_available_runs
```

Comment:
I think this should be renamed as `total_available_runs`.

Reply:
Done!